environment.yml

```diff
@@ -0,0 +1,21 @@
+name: qcodes_2
+channels: 
+ - anaconda
+ - conda-forge
+ - dsdale24
```

Comment:
I think the channels should just be:

```
- defaults
- conda-forge
```

Reply:
The dsdate24 is not used and the preferred name for the default is default. With the setup above I ended up with all packages looking like they are outdated because default has higher priority than anaconda

Reply:
I will make the changes as you suggest and see if I can create a working environment.